ttnn/cpp/ttnn/operations/data_movement/bcast/device/bcast_device_operation.hpp

```diff
@@ -0,0 +1,91 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "ttnn/tensor/tensor.hpp"
+#include "ttnn/operation.hpp"
+
+namespace ttnn {
+
+enum class BcastOpMath { ADD, SUB, MUL };
+enum class BcastOpDim { H, W, HW };
+
+} // namespace ttnn
+
+
+namespace ttnn::operations::data_movement {
+
+struct BcastOperation {
```

Comment:
This should be in bcast.hpp

Reply:
This allows for a clear separation of Device Operation and Operation

Reply:
It’s often the case that some common types are needed, like enums. Can move them to bcast_types.hpp

Reply:
sounds good